### PR TITLE
Add command-line entrypoint

### LINKED ISSUE #52: Could the Distutils check generate an HTML document ?
Hi,

I tested it and this would just be a matter of adding the following at the end of https://github.com/pypa/readme_renderer/blob/master/readme_renderer/integration/distutils.py :
```
else:
    with open('long_description.html', 'w') as long_desc_file:
        long_desc_file.write(markup)
```

Now, are you OK to add this feature ?
I find it very useful to preview the description before uploading a package.

### PR DESCRIPTION
**Pull request by remram44**
Allows you to run `python -m readme_renderer`

Closes #52, closes #53, closes #101.

### CODE REVIEW

**Comment by remram44** [comment]
Will do!

**Comment by remram44** [comment]
Updated

File: readme_renderer/__main__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        description="Renders a .rst README to HTML",
+    )
+    parser.add_argument('input', help="Input README file")
+    parser.add_argument('-o', '--output', help="Output file (default: stdout)")
+    args = parser.parse_args()
+    if args.output:
+        output_file = open(args.output, 'w')
+    else:
+        output_file = sys.stdout
+    input_file = open(args.input)
```
Comment (di) [inline readme_renderer/__main__.py]: You can use [`argparse.FileType`](https://docs.python.org/3/library/argparse.html#filetype-objects) here to eliminate the need for these lines.

Comment (di) [reply readme_renderer/__main__.py]: Specifically, the argument lines can become:

```python
parser.add_argument('input', help="Input README file", type=argparse.FileType('r'))
parser.add_argument('-o', '--output', type=argparse.FileType('w'), default='-', help="Output file (default: stdout)")
```

And then you can just do:

```python
rendered = render(args.input.read(), stream=sys.stderr)
if rendered is None:
    sys.exit(1)
print(rendered, file=args.output)
```

Comment (remram44) [reply readme_renderer/__main__.py]: Oh wow, I didn't know about `type=`. Neat!

**Comment by di** [comment]
@theacodes Can you re-review?

**Comment by di** [comment]
@theacodes Are you cool with this closing out #52 and the other PRs that were attempting to close that issue?

**Comment by theacodes** [comment]
Yes

**Comment by di** [comment]
Thanks @remram44!

**Pull request merged**